Makefile

```diff
@@ -49,23 +49,26 @@ db_sqlite:
 	sqlite3 "$(SQLITE_FILE)" "VACUUM;"
 	$(MAKE) diesel_sqlite
 
+ensure_diesel:
+	@command -v diesel >/dev/null 2>&1 || (echo "diesel_cli not installed, can't continue" && exit 1)
```

Comment:
I think maybe we could require `ensure_diesel` for `db`. It isn't really required, but in the current setup, the containers are started even if diesel_cli isn't installed, and they aren't automatically killed on error